scripts/watermarkstat

```diff
@@ -214,12 +236,15 @@ Examples:
   watermarkstat -t q_shared_multi -c
   watermarkstat -p -t pg_shared
   watermarkstat -p -t q_shared_multi -c
+  watermarkstat -t buffer_pool
```

Comment:
I don't see any changes in show/main.py.
Are there "show command" alias for this? Is the CLI command format decided?